crates/rpc-types-engine/src/jwt.rs

```diff
@@ -120,6 +124,30 @@ impl JwtSecret {
         }
     }
 
+    /// Tries to load a [`JwtSecret`] from the specified file path.
+    /// I/O or secret validation errors might occur during read operations in the form of
+    /// a [`JwtError`].
+    pub fn from_file(fpath: &Path) -> Result<Self, JwtError> {
+        let hex = fs::read_to_string(fpath)?;
+        let secret = JwtSecret::from_hex(hex)?;
+        Ok(secret)
+    }
+
+    /// Creates a random [`JwtSecret`] and tries to store it at the specified path. I/O errors might
+    /// occur during write operations in the form of a [`JwtError`]
+    pub fn try_create(fpath: &Path) -> Result<Self, JwtError> {
```

Comment:
function name should indicate that the created token is random. function should be generic over `Rng` trait

Reply:
updated the function name. 

> function should be generic over Rng trait

Do you still need me to update the `random` method to be more generic? I just ported the code over from Reth and it was not something that was yet supported there, so not sure if this is in scope for this change.

Reply:
no, i think this is fine. if we're not changing the internals of `random()` let's just leave this as is